src/client.py

```diff
@@ -4629,6 +4632,21 @@ def pubcheck_linked(op, api_inst, pargs):
 
         return EXIT_OK
 
+def hotfix_cleanup(op, api_inst, pargs):
+        try:
+                api_inst.hotfix_origin_cleanup()
+        except api_errors.ImageLockedError as e:
+                error(e)
+                return EXIT_LOCKED
+        except api_errors.ImageMissingKeyFile as e:
+                error(e)
+                return EXIT_EACCESS
+        except api_errors.UnprivilegedUserError as e:
+                error(e)
+                return EXIT_OOPS
+
```

Comment:
does this mean if there is another sort of exception you will return happy ? or will it be passed up ?

Reply:
It will be passed up and might end up with a stack trace for the caller.
These are the same exceptions that the internal `pubcheck-linked` command catches, apart from UnprivilegedUserError which is just for this one because I try to acquire a privileged lock on the image.